lib/dynflow/rails.rb

```diff
@@ -38,8 +38,8 @@ def initialize!
       init_world.tap do |world|
         @world = world
         config.run_on_init_hooks(false, world)
+        config.increase_db_pool_size(world) if config.increase_db_pool_size?
```

Comment:
This is checked inside of the method already.
```suggestion
        config.increase_db_pool_size(world)
```